src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Storage {
+    public static final List<FruitTransaction> transactions = new ArrayList<>();
+    public static final Map<String, Integer> fruitMap = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<String, Integer> fruitMap = new HashMap<>();
```

no need to save transations here